components/file_system/src/io_stats/mod.rs

```diff
@@ -54,7 +55,7 @@ mod tests {
         let _ths = (0..8)
             .map(|_| {
                 let tx_clone = tx.clone();
-                std::thread::Builder::new().spawn(move || {
+                std::thread::spawn(move || {
```

Comment:
should we disallow `std::thread::spawn`?

Reply:
I think it is not necessary. Since this pr actually adds methods to set hooks for thread pool builder and we should deny regular set-hook methods only.